content/hi/docs/setup/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: शुरू करना
+main_menu: true
+weight: 20
+content_type: concept
+no_list: true
+card:
+  name: setup
+  weight: 20
+  anchors:
+  - anchor: "#सीखने-का-वातावरण"
+    title: सीखने का वातावरण
+  - anchor: "#प्रोडक्शन-वातावरण"
+    title: प्रोडक्शन वातावरण
+---
+
+<!-- overview -->
+
+यह भाग कुबेरनेट्स को सेटअप करने और चलाने के विभिन्न तरीकों को सूचीबद्ध करता है। जब आप कुबेरनेट्स सेटअप करते हैं, तो इसके आधार पर एक इंस्टॉलेशन प्रकार चुनें: रखरखाव में आसानी, सुरक्षा, नियंत्रण, उपलब्ध संसाधन और क्लस्टर के संचालन और प्रबंधन के लिए आवश्यक विशेषज्ञता।
+
+कुबेरनेट्स क्लस्टर को स्थानीय मशीन पर, क्लाउड में, या अपने स्वयं के डेटासेंटर में डिप्लॉय करने के लिए आप [कुबेरनेट्स डाउनलोड](/releases/download/) कर सकते हैं।
+
+यदि आप कुबेरनेट्स क्लस्टर को स्वयं संभालना नहीं चाहते हैं, तो आप एक संचालन सेवा चुन सकते हैं, जैसे ये [प्रमाणित प्लेटफॉर्म](/hi/docs/setup/production-environment/turnkey-solutions/)।
```

Comment:
Does https://k8s.io/hi/docs/setup/production-environment/turnkey-solutions/ already exist as a localized page in branch _dev-1.22-hi.1_ ? If not, I would link to `/docs/setup/production-environment/turnkey-solutions/`

Reply:
It doesn't exist at the moment but the PR to add it is under review. It will be a part of `dev-1.22-hi.1` by the time we merge it to `main`.

Reply:
so, do i need to change anything?